tests/runtime/file-output

```diff
@@ -0,0 +1,19 @@
+NAME printf to file
+RUN bpftrace -v -e 'i:ms:10 { printf("%s %d\n", "SUCCESS", 1); exit() }' -o "out.tmp" >/dev/null; cat "out.tmp"; rm "out.tmp"
```

Comment:
I'd suggest writing to a file like `/tmp/bpftrace-file-output-test` to avoid  writing to the regular file system and reduce the chance of that file already existing

Reply:
woops